CONTRIBUTING.rst

```diff
@@ -133,6 +133,14 @@ premature merging of the PR.
 
 .. _certain files: .github/labels.yml#L12-L13
 
+Predicted Risk
+~~~~~~~~~~~~~~
+PRs that touch certain files will be automatically flagged with a "Predicted Risk" label,
```

Comment:
```suggestion
PRs that touch certain files will be automatically flagged with a "Risk" label,
```
(if you don't take the suggestion above about dropping `Expected` then this should change to `Expected Risk`)